scene/animation/animation_tree_player.cpp

```diff
@@ -401,6 +401,7 @@ void AnimationTreePlayer::_notification(int p_what) {
 
 		case NOTIFICATION_ENTER_TREE: {
 
+			WARN_PRINTS("AnimationTreePlayer has been deprecated. Use AnimationTree instead.");
```

Comment:
You can use `ERR_EXPLAIN` + `WARN_DEPRECATED` instead, which will only show the deprecation warning once for session, instead of spamming each time an ATP enters the tree.

Reply:
Done.